src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CollectionPropertiesShouldBeReadOnlyTests.cs

```diff
@@ -431,5 +431,39 @@ class A
 }
 ");
         }
+
+        [Fact, WorkItem(4461, "https://github.com/dotnet/roslyn-analyzers/issues/4461")]
+        public async Task CA2227_CSharp_InitPropertyRecord()
```

Comment:
Test are failing because some type is not imported, I suspect there might be a need to update the test sdk but I am not entirely sure.

@sharwell may I ask your help?

Reply:
FYI the error is `error CS0518: Predefined type 'System.Runtime.CompilerServices.IsExternalInit' is not defined or imported`